packages/popover/README.md

```diff
@@ -59,22 +59,22 @@ The popover component will be automatically positioned relative to its nearest p
+| `spacing`          | `number`                                                                                       | Specifies the amount of spacing (in pixels) between the trigger element and the content element.                                                                                                                         | `10`       |
+| `className`        | `string`                                                                                       | Classname to apply to popover-content container                                                                                                                                                                          |            |
+| `adjustOnMutation` | `boolean`                                                                                      | Should the Popover auto adjust its content when the DOM changes (using MutationObserver).                                                                                                                                | `false`    |
+| `onClick`          | `function`                                                                                     | Function that will be called when popover content is clicked.                                                                                                                                                            |            |
 
 ## Advanced Use Case
 
-| Prop    | Type     | Description                                                                                                                                                                                         | Default |
-| ------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- |
-| `refEl` | `object` | You can supply a `refEl` prop, if you do not want the popover to be positioned relative to it's nearest parent. Ref to the element to which the popover component should be positioned relative to. | `null`  |
+| Prop    | Type          | Description                                                                                                                                                                                         | Default |
+| ------- | ------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- |
+| `refEl` | `HTMLElement` | You can supply a `refEl` prop, if you do not want the popover to be positioned relative to it's nearest parent. Ref to the element to which the popover component should be positioned relative to. | `null`  |
 
 _Any other properties will be spread on the popover-content container_
```

Comment:
is there going to be a follow-up item for the remainder of these notes to become a `...` prop?

Reply:
I should be able to remove this entirely because PopoverProps doesn't extend anything, right?

Reply:
I think that's probably more of an oversight. Since this comment exists, I assume the intent was to support it. It'd be a breaking change to remove it so if we do we should probably do it separately and change the implementation to not spread the rest props.

Reply:
Okay, just added tot he table without the typing of the rest props